src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinValueInstantiator.kt

```diff
@@ -26,6 +26,7 @@ import kotlin.reflect.jvm.javaType
 internal class KotlinValueInstantiator(
     src: StdValueInstantiator,
     private val cache: ReflectionCache,
```

Comment:
This field is currently no longer referenced from within the `KotlinValueInstantiator`.
I'm going to leave it as is until the cache handling question I'm asking in another comment is finalized.

Reply:
This comment also resolves.
https://github.com/FasterXML/jackson-module-kotlin/pull/439#discussion_r632773724